spec.html

```diff
@@ -14144,18 +14144,25 @@ <h2>Syntax</h2>
     <emu-clause id="sec-assignment-operators-static-semantics-early-errors">
       <h1>Static Semantics: Early Errors</h1>
       <emu-grammar>AssignmentExpression : LeftHandSideExpression `=` AssignmentExpression</emu-grammar>
+      <p>If |LeftHandSideExpression| is an |ObjectLiteral| or an |ArrayLiteral|, the following Early Error rules are applied:</p>
       <ul>
         <li>
-          It is a Syntax Error if |LeftHandSideExpression| is either an |ObjectLiteral| or an |ArrayLiteral| and |LeftHandSideExpression| is not covering an |AssignmentPattern|.
+          It is a Syntax Error if |LeftHandSideExpression| is not covering an |AssignmentPattern|.
         </li>
         <li>
-          It is an early Syntax Error if |LeftHandSideExpression| is neither an |ObjectLiteral| nor an |ArrayLiteral| and AssignmentTargetType of |LeftHandSideExpression| is not ~simple~.
+          All Early Error rules for |AssignmentPattern| and its derived productions also apply to the |AssignmentPattern| that is covered by |LeftHandSideExpression|.
+        </li>
+      </ul>
+      <p>If |LeftHandSideExpression| is neither an |ObjectLiteral| nor an |ArrayLiteral|, the following Early Error rule is applied:</p>
+      <ul>
+        <li>
+          It is a Syntax Error if AssignmentTargetType of |LeftHandSideExpression| is not ~simple~.
         </li>
       </ul>
       <emu-grammar>AssignmentExpression : LeftHandSideExpression AssignmentOperator AssignmentExpression</emu-grammar>
       <ul>
         <li>
-          It is an early Syntax Error if AssignmentTargetType of |LeftHandSideExpression| is not ~simple~.
+          It is a Syntax Error if AssignmentTargetType of |LeftHandSideExpression| is not ~simple~.
```

Comment:
Seems like there are only three other matches for `early Syntax Error` in the spec—shall we update those right away too?